### ISSUE TITLE
[BottomAppBar] Add a11y guide to Bottom App Bar readme

### ISSUE THREAD

**Issue by jmdetloff** [material-components/material-components-ios#4284]
Outline the need to set appropriate labels and ensure proper contrast.

**Comment by jverkoey** [comment]
@codeman7 Outline what needs to be done before we can write this doc.

**Comment by jverkoey** [comment]
@codeman7 Please link to blocking issues.

**Comment by romoore** [comment]
@jverkoey I think the description is pretty straightforward - accessibility labels and color theming. Is there something else that is missing before this issue can be worked?

**Comment by jverkoey** [comment]
@codeman7 had mentioned there were issues that were blocking us from landing the guides. Can you elaborate on that @codeman7 ?

**Comment by codeman7** [comment]
I think we are good just some things people have to be aware of. I'll out line them in the doc.
